visualization/app/codeCharta/util/customViewHelper.ts

```diff
@@ -0,0 +1,288 @@
+"use strict"
+import { ColorRange, LocalStorageCustomViews, stateObjectReplacer, stateObjectReviver } from "../codeCharta.model"
+import { CustomViewItemGroup } from "../ui/customViews/customViews.component"
+import { CustomView, CustomViewMapSelectionMode, CustomViewsDownloadFile, ExportCustomView } from "../model/customView/customView.api.model"
+import { CustomViewFileStateConnector } from "../ui/customViews/customViewFileStateConnector"
+import { FileNameHelper } from "./fileNameHelper"
+import { FileDownloader } from "./fileDownloader"
+import { setState } from "../state/store/state.actions"
+import { setColorRange } from "../state/store/dynamicSettings/colorRange/colorRange.actions"
+import { setMargin } from "../state/store/dynamicSettings/margin/margin.actions"
+import { setCamera } from "../state/store/appSettings/camera/camera.actions"
+import { Vector3 } from "three"
+import { setCameraTarget } from "../state/store/appSettings/cameraTarget/cameraTarget.actions"
+import { StoreService } from "../state/store.service"
+import { ThreeCameraService } from "../ui/codeMap/threeViewer/threeCameraService"
+import { ThreeOrbitControlsService } from "../ui/codeMap/threeViewer/threeOrbitControlsService"
+import { CodeChartaStorage } from "./codeChartaStorage"
+
+export const CUSTOM_VIEW_FILE_EXTENSION = ".cc.config.json"
+const CUSTOM_VIEWS_LOCAL_STORAGE_VERSION = "1.0.0"
+const CUSTOM_VIEWS_DOWNLOAD_FILE_VERSION = "1.0.0"
+export const CUSTOM_VIEWS_LOCAL_STORAGE_ELEMENT = "CodeCharta::customViews"
```

Comment:
This must not be renamed because Custom Configs that are stored under CodeCharta::customConfigs in the local storage won't work anymore. So please keep the old key.